.changeset/chatty-tables-attend.md

```diff
@@ -0,0 +1,12 @@
+---
+'@launchpad-ui/inline-edit': minor
+'@launchpad-ui/core': patch
+---
+
+Add `inline-edit` package to display and allow inline editing of a form elements:
+
+- Use props `defaultValue` and `onSave` to handle state management of the value to edit
```

Comment:
This means this component cannot be "controlled", right?

Are there use  cases where we might want it to be controlled too?

Reply:
Yeah having controlled can be useful! Will add it.